org.videolan.VLC.json

```diff
@@ -0,0 +1,671 @@
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://caca.zoy.org/raw-attachment/wiki/libcaca/libcaca-0.99.beta19.tar.gz",
+          "sha256": "128b467c4ed03264c187405172a4e83049342cc8cc2f655f53a2d0ee9d3772f4"
+        }
+      ]
+    },
+    {
+      "name": "aalib",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/aa-project/aa-lib/1.4rc5/aalib-1.4rc5.tar.gz",
+          "sha256": "fbddda9230cf6ee2a4f5706b4b11e2190ae45f5eda1f0409dc4f99b35e0a70ee"
+        }
+      ]
+    },
+    {
+      "name": "lirc",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/lirc/LIRC/0.9.4c/lirc-0.9.4c.tar.bz2",
+          "sha256": "8974fe5dc8eaa717daab6785d2aefeec27615f01ec24b96d31e3381b2f70726a"
+        }
+      ]
+    },
+    {
+      "name": "fftw",
+      "config-opts": [
+        "--enable-shared",
+        "--enable-threads",
+        "--enable-openmp"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.fftw.org/fftw-3.3.5.tar.gz",
+          "sha256": "8ecfe1b04732ec3f5b7d279fdb8efcad536d555f9d1e8fabd027037d45ea8bcf"
+        }
+      ]
+    },
+    {
+      "name": "chromaprint",
+      "cmake": true,
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://bitbucket.org/acoustid/chromaprint/downloads/chromaprint-1.2.tar.gz",
+          "sha256": "822b8949a322ac04c6f8a3ed78f5e689bcc493c6ca0972bf627c913c8430031a"
+        }
+      ]
+    },
+    {
+      "name": "libshout",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.xiph.org/releases/libshout/libshout-2.4.1.tar.gz",
+          "sha256": "f3acb8dec26f2dbf6df778888e0e429a4ce9378a9d461b02a7ccbf2991bbf24d"
+        }
+      ]
+    },
+    {
+      "name": "libupnp",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/pupnp/pupnp/libUPnP%201.6.20/libupnp-1.6.20.tar.bz2",
+          "sha256": "ee3537081e3ea56f66ada10387486823989210bc98002f098305551c966e3a63"
+        }
+      ]
+    },
+    {
+      "name": "dbus-glib",
+      "config-opts": [
+        "--enable-static=no",
+        "--enable-bash-completion=no"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.108.tar.gz",
+          "sha256": "9f340c7e2352e9cdf113893ca77ca9075d9f8d5e81476bf2bf361099383c602c"
+        }
+      ]
+    },
+    {
+      "name": "dbus-python",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.4.tar.gz",
+          "sha256": "e2f1d6871f74fba23652e51d10873e54f71adab0525833c19bad9e99b1b2f9cc"
+        }
+      ]
+    },
+    {
+      "name": "avahi",
```

Comment:
I believe avahi doesn't work in the sandbox, at least not without extra work. If optional maybe disable that for now?

See also flatpak/flatpak#348